dds/src/SystemHandle.cpp

```diff
@@ -34,6 +33,7 @@ namespace dds{
 namespace {
 
 // This function patches the problem of dynamic types, which do not admit '/' in their type name.
+/*
```

Comment:
This saved us about the type name from ros2 with `/`. How do you solve this?

Reply:
I didn't care about it, but I'm sure it is necessary. That's because I leave it commented.

Reply:
ok